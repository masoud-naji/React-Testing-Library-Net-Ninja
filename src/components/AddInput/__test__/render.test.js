import TestRenderer from "react-test-renderer";

function Link(props) {
  return <a href={props.page}>{props.children}</a>;
}



it("Matches the snapshot facebook Link", () => {
  const tree = TestRenderer.create(<Link page="https://www.facebook.com/">Facebook</Link>).toJSON();
  expect(tree).toMatchSnapshot();
});