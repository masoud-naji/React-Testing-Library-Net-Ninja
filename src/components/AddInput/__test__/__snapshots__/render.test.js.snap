// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Matches the snapshot 1`] = `<div />`;

exports[`Matches the snapshot face 1`] = `
<a
  href="https://www.facebook.com/"
>
  Facebook
</a>
`;
